### DISCUSSION TITLE
[功能提议]新的图表

### DISCUSSION THREAD

**Discussion by lxdklp** [PBH-BTN/PeerBanHelper#422] [讨论]
我非常喜欢PeerBanHelper的图表统计功能，请问是否能加上连接的客户端统计(如图，PeerBanHelper能获取到客户端信息)
![屏幕截图 2024-09-02 195312](https://github.com/user-attachments/assets/fec1e6ab-a1bf-46e2-ba28-9aa76a9aee93)  
以及能否在流量统计加上分别是由什么种子上传的流量，上传了多少g流量
![屏幕截图 2024-09-03 191855](https://github.com/user-attachments/assets/c93c041e-d65e-4d03-8252-4a16da389f70)

**Comment by Ghost-chu** [comment]
> 我非常喜欢 PeerBanHelper 的图表统计功能，请问是否能加上连接的客户端统计(如图，PeerBanHelper 能获取到客户端信息)

这个应该已经有了，不过是 PeerID 版本

> 以及能否在流量统计加上分别是由什么种子上传的流量，上传了多少g流量

此功能已存在，但目前前端没适配。

**Reply by lxdklp** [reply]
我知道有peer ID功能，但是个人觉得显示客户端名称更加直观。而下载器的流量统计是总计的流量，我的想法是pbh能提供一个按日统计的界面

**Comment by Gaojianli** [comment]
> 2024年9月18日 21:03，Ghost_chu ***@***.***> 写道：
> 
> 
> 我非常喜欢 PeerBanHelper 的图表统计功能，请问是否能加上连接的客户端统计(如图，PeerBanHelper 能获取到客户端信息)
> 
> 这个应该已经有了，不过是 PeerID 版本
> 
> 以及能否在流量统计加上分别是由什么种子上传的流量，上传了多少g流量
> 
> 此功能已存在，但目前前端没适配。
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/PBH-BTN/PeerBanHelper/discussions/422#discussioncomment-10681510>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AESYDLR5W2EPFKV775RK3PDZXF2ZDAVCNFSM6AAAAABNR5ES22VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRYGE2TCMA>.
> You are receiving this because you are subscribed to this thread.
> 
种子数量不可控，很难做到比较好的显示效果

**Comment by Ghost-chu** [comment]
种子流量下载器自己就有统计的呀

![image](https://github.com/user-attachments/assets/e0880ad3-aed4-4fad-9f4e-037440e7eb4e)

**Comment by Ghost-chu** [comment]
![image](https://github.com/user-attachments/assets/0c98d177-82fb-4ad2-ae4c-913d10893b63)
